lib/class-wp-json-terms-controller.php

```diff
@@ -212,4 +212,43 @@ protected function check_valid_taxonomy( $taxonomy ) {
 		}
 	}
 
+	/**
+	 * Get the schema for a term
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		return array(
+			'title'                => 'Term',
+			'type'                 => 'object',
```

Comment:
@danielbachhuber could you give an example when `type` is not `object`? I guess collection endpoints?